src/datagen/utils.ts

```diff
@@ -121,7 +29,7 @@ function initalizeUsersAndSessions({
   return sessions;
 }
 
-function getNormalizedWeights(categories: Category[]) {
+export function getNormalizedWeights(categories: Category[]) {
```

Comment:
Internally, we have a rule which says that every exported function must have a JSDoc comment describing what it does.  By marking this with export, this method is now part of the public API of this module.

Reply:
Do you know if there is a ESLint rule for that? We are currently using [gts (Google TypeScript Style)](https://github.com/google/gts) and think it would be great if I can add the rule on top of it.

Reply:
It looks like there isn't a specific rule for this one.  I suspect the internal TS team wrote something themselves.